packages/@lwc/template-compiler/src/__tests__/config.spec.ts

```diff
@@ -77,7 +77,7 @@ describe('customRendererConfig normalization', () => {
                 },
             })
         ).toThrowErrorMatchingInlineSnapshot(
-            `"LWC1150: customRendererConfig contains duplicate entry for use element tag"`
+            `[Error: LWC1150: customRendererConfig contains duplicate entry for use element tag]`
```

Comment:
It's much easier to change the test here than make it work with vitest hopefully this is fine.